CIP-xxxx/README.md

```diff
@@ -0,0 +1,111 @@
+---
+CPS: ????
+Title: SRV registry
+Status: Active
+Category: Network
+Authors:
+  - Marcin Szamotulski <marcin.szamotulski@iohk.io>
+Implementors: N/A
+Discussions: 
+  - [PR](https://github.com/cardano-foundation/CIPs/pull/xxx)
+Created: 2025-04-22
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP defines the procedure for assigning SRV prefixes for Cardano applications (like `cardano-node`, `mithril`, etc.).
+
+## Motivation
+
+The **Cardano Ledger** allows the use of SRV records in the **SPO** registration certificate.
```

Comment:
Can you link to some resource explaining this or if not available include an example how this is typically done?

Reply:
`cardano-cli` supports it via the `--multi-host-pool-relay` flag, see: https://developers.cardano.org/docs/operate-a-stake-pool/register-stake-pool/